frontend/src/contexts/FederationContext.tsx

```diff
@@ -107,10 +107,10 @@ export const FederationContextProvider = ({
     // On bitcoin network change we reset book, limits and federation info and fetch everything again
     const newFed = initialFederationContext.federation;
     newFed.registerHook('onFederationReady', () => {
-      setCoordinatorUpdatedAt(new Date().toISOString());
+      setFederationUpdatedAt(new Date().toISOString());
     });
     newFed.registerHook('onCoordinatorUpdate', () => {
-      setFederationUpdatedAt(new Date().toISOString());
+      setCoordinatorUpdatedAt(new Date().toISOString());
```

Comment:
UpdatedAts where being triggered the other way around 😓 Fortunately `federationUpdatedAt` where not much used, that meas that in the worst case scenario components now will refresh more frequently. Anyways I would say to do a good check of everything for this PR.

Reply:
it's indeed likely that with this change some minor bugs on the list will be fixed